app/views/safetyDiagnosticTool/sdt-layout.html

```diff
@@ -0,0 +1,25 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Digital Prison Reporting MI Prototype - Safety Diagnostic Tool
+{% endblock %}
+
+{% block subHeader %}
+<div class="govuk-width-container ">
+  {{ govukBreadcrumbs({
+    items: [
+      { text: "Home", href: "/" },
+      { text: "Safety Diagnostic Tool", href: "/safety-diagnostic-tool/" }
+    ]
+  }) }}
+</div>
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-width-container ">
+    {% block reportBody %}{% endblock %}
```

Comment:
I'm not sure "reportBody" is quite right here (as there's no reports). Maybe something more generic like "contentBody"?